### DISCUSSION TITLE
Is there a fill() in Form objects?

### DISCUSSION THREAD

**Discussion by jeffchown** [livewire/livewire#6688] [1. Help] [answered]
Hello,

For `Form` objects that are passed a Model, e.g. a `$post` for an update, is there a quick way to assign the model's attributes to their equivalent public properties in the Form? (e.g. equivalent of `$this->fill([...])` in a regular Livewire Component?)

Thanks!

**Comment by joshhanley** [comment]
@jeffchown hmm not sure, maybe not. PRs are welcome! 🙂

**Reply by jeffchown** [reply]
Thanks @joshhanley 👍 Just made my first (ever) PR (https://github.com/livewire/livewire/pull/6707)

**Reply by joshhanley** [reply, marked as answer]
@jeffchown nice, great job! Thanks for the PR 😁

**Reply by jeffchown** [reply]
Happy to contribute.  Thanks, @joshhanley!
